catalogue_graph/src/adapters/ebsco/transformers/subjects.py

```diff
@@ -99,7 +99,19 @@ def subdivision_concepts_648_650_651(field: Field) -> Generator[Concept]:
 
 def extract_subjects(record: Record) -> list[Subject]:
     return non_empty(
-        extract_subject(field) for field in record.get_fields(*SUBJECT_FIELDS)
+        extract_subject(field)
+        for field in record.get_fields(*SUBJECT_FIELDS)
+        if is_subject_to_keep(field)
+    )
+
+
+def is_subject_to_keep(field: Field) -> bool:
+    return field.indicators is not None and (
+        field.indicators.second in ["0", "2"]
+        or (
+            field.indicators.second == "7"
+            and field.get("2") in ["local", "homoit", "indig", "enslv"]
```

Comment:
Maybe link or reference the Scala implementation of this filter - i've seen it hanging about somewhere!